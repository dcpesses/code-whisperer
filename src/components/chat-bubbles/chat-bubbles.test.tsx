import {render} from '@testing-library/react';
import ChatBubbles from './index';

describe('ChatBubbles', () => {
  test('Should render as expected', () => {
    const {container} = render(<ChatBubbles />);
    expect(container).toMatchSnapshot();
  });
});
