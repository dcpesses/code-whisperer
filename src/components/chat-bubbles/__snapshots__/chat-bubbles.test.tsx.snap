// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ChatBubbles > Should render as expected 1`] = `
<div>
  <div
    class="chat-bubbles"
  >
    <div
      class="stars"
      id="stars1"
    />
    <div
      class="stars"
      id="stars2"
    />
    <div
      class="stars"
      id="stars3"
    />
    <div
      class="chats"
      id="chats1"
    >
      
    </div>
    <div
      class="chats"
      id="chats2"
    >
      
    </div>
  </div>
</div>
`;
